kaldb/src/test/java/com/slack/kaldb/preprocessor/PreprocessorValueMapperTest.java

```diff
@@ -31,7 +31,7 @@ public void shouldValueMapApiLog() {
             .build();
 
     ValueMapper<byte[], Iterable<Trace.Span>> valueMapper =
-        PreprocessorValueMapper.byteArrayToTraceSpans("api_log");
+        PreprocessorValueMapper.byteArrayToTraceSpans("api_log", null);
```

Comment:
Instead of null, please pass in a ` meterRegistry = new SimpleMeterRegistry();` in these tests.